src/session.rs

```diff
@@ -521,6 +565,24 @@ mod tests {
         assert_eq!(output_tensor[1], 6.0);
     }
 
+    #[test]
+    fn test_run_metadata() {
+        let (session, x_operation, y_operation) = create_session();
+        let mut x = <Tensor<f32>>::new(&[2]);
+        x[0] = 2.0;
+        x[1] = 3.0;
```

Comment:
I know we don't have examples, but you can replace the previous three lines with the shorter and less error-prone (although slightly odd):

```
let x = Tensor::<f32>::from(&[2.0, 3.0][..]);
```

(The weird `[..]` is because we have a `From` impl for slices but not arrays, and for some reason the compiler doesn't catch on that `&[2.0, 3.0]` can be treated as a slice as well as a reference to an array.  This is a shorthand for treating it as a slice.)

Reply:
Thanks! Replaced with a slice.